internal/resources/metal/organization/resource_test.go

```diff
@@ -1,14 +1,16 @@
-package equinix
+package organization_test
```

Comment:
There's a test failure in the project tests as well that looks related to these changes:
```
=== NAME  TestAccMetalProject_organization
    resource_test.go:361: Step 1/2 error: Error running apply: exit status 1
        
        Error: Provider produced inconsistent result after apply
        
        When applying changes to equinix_metal_organization.test, provider
        "provider[\"registry.terraform.io/hashicorp/equinix\"]" produced an
        unexpected new value: .description: was null, but now cty.StringVal("").
        
        This is a bug in the provider, which should be reported in the provider's own
        issue tracker.
```
(Note: this shouldn't be resolved with changes to the project test; this failure is an indication that migrating from SDK to framework changed the resource behavior in some way, and that behavior change needs to be fixed.)